modules/infrastructure-moving-router.adoc

```diff
@@ -14,10 +14,10 @@ is displayed to a worker node.
 
 .Procedure
 
-. View the `clusteringress` Custom Resource for the router Operator:
+. View the `IngressController` Custom Resource for the router Operator:
 +
 ----
-$ oc get clusteringress default -n openshift-ingress-operator -o yaml <1>
+$ oc get ingresscontroller default -n openshift-ingress-operator -o yaml <1>
```

Comment:
@Miciah, the CR for the router's changed, and I'm not sure if the instructions for moving it to an infrastructure node is right anymore. Will you PTAL?

Reply:
I'm confused by the change.  It looks like you started updating the existing example and then added a another version that supersedes the old version.  You can delete https://github.com/openshift/openshift-docs/blob/0947083c5fd006e36ba64c11f2f36203d3446d29/modules/infrastructure-moving-router.adoc#L30-L57 (which has the updated `kind` but incorrect `selfLink` and old `spiVersion`, `finalizers`, `spec`, and `status`) and keep https://github.com/openshift/openshift-docs/blob/0947083c5fd006e36ba64c11f2f36203d3446d29/modules/infrastructure-moving-router.adoc#L59-L81 (which is correct), and then that example will be correct.

See my other comments for a couple other minor issues.  Also, the `oc get pod` output at the end of the document is funny—I don't get `IP`, `NODE`, or `NOMINATED NODE` columns when I run the command.  With those changes, the instructions should be fine.